src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -84,21 +87,22 @@
     },
     {
       "what": [
-        "{APP_NAME_ADMIN} user view not properly updated after 'Confirm email address' was clicked in {APP_NAME_ADMIN}."
+        "{APP_NAME_ADMIN} user view not properly updated after 'Confirm email address' was clicked in {APP_NAME_ADMIN}.",
+        "The user view was just staying empty, instead of being populated with the user data."
```

Comment:
Can we squash the two together and get rid of the ambiguous `not properly updated` part?